app/views/devise/passwords/new.html.slim

```diff
@@ -4,12 +4,14 @@
   .medium-6.medium-centered.large-4.large-centered.columns
     .row.column.log-in-form
       h2.text-center
-        | Forgot your password?
+        | Forgot Your Password?
       = simple_form_for(resource, as: resource_name, url: password_path(resource_name), html: { method: :post }) do |f|
         = f.error_notification
         .form-inputs
           = f.input :email, required: true, autofocus: true
+        br
```

Comment:
I understand what you're saying, separating styling and mark-up.
But the `<br>` tag is an HTML element that creates that line break.
Do you think it'll be better to add a _.gap_ class, for something that is handled by the _single line break tag_?

<img width="787" alt="screen shot 2017-04-28 at 9 37 46 am" src="https://cloud.githubusercontent.com/assets/18417244/25519399/eaf271e0-2bf7-11e7-82f0-fe1d97721635.png">